client/my-sites/promote-post-i2/components/campaigns-total-stats/index.tsx

```diff
@@ -0,0 +1,34 @@
+import { translate } from 'i18n-calypso';
+import { formatNumber } from '../../utils';
+import './style.scss';
+
+type Props = {
+	totalImpressions?: number;
+	totalClicks?: number;
+	outerContainerClass?: string;
+};
+
+export default function CampaignsTotalStats( {
+	totalImpressions,
+	totalClicks,
+	outerContainerClass,
+}: Props ) {
+	return (
+		<div className={ outerContainerClass }>
+			<div className="campaigns-total-stats__container">
+				<div className="campaigns-total-stats__item">
+					<div className="campaigns-total-stats__label">
+						{ translate( 'Total people reached' ) }
+					</div>
+					<div className="campaigns-total-stats__result">
+						{ formatNumber( totalImpressions || 0 ) }
+					</div>
+				</div>
+				<div className="campaigns-total-stats__item">
+					<div className="campaigns-total-stats__label">{ translate( 'Total visitors' ) }</div>
```

Comment:
We are using clicks instead of visitors here. The designs are not fully updated. 

We wanted to use clicks to differentiate with the visitors metric. Not all clicks end up being visitors.

Reply:
fixed.. thanks